<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::ViewIdle</class>
 <widget class="QWidget" name="ns::ViewIdle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>567</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YesPo Service Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana,&quot;微软雅黑&quot;,&quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}
#widgetTop {
/*	border-right: 1px solid #57a3d3;*/
/*	border: 0px;*/
	/*border-image: url(:/images/ui/bg_processing.png);*/
/*	border-bottom-right-radius: 3px;*/
}

#widgetTop #labelWaiting {
	color: rgb(255, 255, 255, 80%);
	font-size: 25px;	

	font-weight: bold;
}

#widgetDuration #labelDuration {
	color: rgb(255, 255, 255, 85%);
	font-size: 95px;
	font-weight: bold;
}

#widgetDuration #labelIntroduce {
	color: rgb(255, 255, 255, 70%);
	font-size: 32px;
	font-weight: bold;
}

#widgetDuration #labelCode {
	color: rgb(255, 255, 255, 60%);
	font-size: 22px;

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ns::CustomFadeWidget" name="widgetTop" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widgetDuration" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>50</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetWaiting" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="ns::CustomFontLabel" name="labelWaiting">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Waiting call</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignHCenter|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="ns::CustomFontLabel" name="labelDuration">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>136</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>136</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string notr="true">10:10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignHCenter|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ns::CustomFontLabel" name="labelIntroduce">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>47</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>47</height>
              </size>
             </property>
             <property name="text">
              <string>Good morning, Tony cheng</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ns::CustomFontLabel" name="labelCode">
             <property name="text">
              <string>1111111111</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignHCenter|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns::CustomFadeWidget</class>
   <extends>QWidget</extends>
   <header>customfadewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ns::CustomFontLabel</class>
   <extends>QLabel</extends>
   <header>customfontlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections/>
</ui>
